xarray/plot/facetgrid.py

```diff
@@ -142,6 +142,8 @@ def __init__(self, data, col=None, row=None, col_wrap=None,
 
         # Set the subplot kwargs
         subplot_kws = {} if subplot_kws is None else subplot_kws
+        if transform is not None and 'projection' not in subplot_kws:
+            subplot_kws['projection'] = transform
```

Comment:
I think we should only make this assumption if the projection is a `cartopy.crs` object. That would be easy to test here.